docs/examples/website-to-api/web_scraper_lib.py

```
@@ -0,0 +1,397 @@
+        Returns:
+            True if saved successfully
+        """
+        try:
+            model_config = ModelConfig(provider, api_token)
+            config_path = os.path.join(self.models_dir, f"{model_name}.json")
+            
+            with open(config_path, "w") as f:
+                json.dump(model_config.to_dict(), f, indent=2)
+            
+            print(f"Model configuration saved: {model_name}")
+            return True
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)